app/lib/actions/ui.js

```diff
@@ -1,6 +1,8 @@
+import { php_escapeshellcmd } from 'php-escape-shell';
```

Comment:
I don't like it either, believe me, I would prefer that the author had choose a different name. But why set an alias? There is no name conflicts and the original name is self-explicative. Keeping the original name simply helps code clarity imho, that's why i kept it.